sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -690,6 +690,7 @@ class SparkSession private(
    * @since 2.0.0
    */
   def stop(): Unit = {
+    streams.stopAllQueries()
```

Comment:
This one actually doesn't stop queries in the child `SparkSession`s. You can register a SparkListener when creating StreamingQueryManager to monitor `SparkListenerApplicationEnd` and stop queries.